wkcuber/poetry.lock

```diff
@@ -487,17 +487,17 @@ python-versions = ">=3.5"
 
 [[package]]
 name = "imagecodecs"
-version = "2020.12.24"
+version = "2021.11.20"
 description = "Image transformation, compression, and decompression codecs"
 category = "main"
 optional = false
 python-versions = ">=3.7"
 
 [package.dependencies]
-numpy = ">=1.15.1"
+numpy = ">=1.16.5"
```

Comment:
This just specifies that the selected imagecodecs version has this requirement and is fine :+1: